web/packages/teleport/src/lib/term/terminal.ts

```diff
@@ -98,6 +122,21 @@ export default class TtyTerminal {
     window.addEventListener('resize', this._debouncedResize);
   }
 
+  fallbackToCanvas() {
+    logger.info('WebGL context lost. Falling back to canvas');
+    this._webglAddon.dispose();
```

Comment:
seems safer since it's initially undefined

```suggestion
    logger.info('WebGL context lost. Falling back to canvas');
    this._webglAddon?.dispose();
```

Reply:
absolutely right. ty